streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamPrintTest.java

```diff
@@ -82,9 +83,42 @@ public void testPrintKeyValueWithName() {
             driver.process(topicName, record.key, record.value);
         }
         printWriter.flush();
-        final String[] flushOutDatas = new String(byteOutStream.toByteArray(), Charset.forName("UTF-8")).split("\n");
+        final String[] flushOutDatas = new String(byteOutStream.toByteArray(), Charset.forName("UTF-8")).split("\\r*\\n");
         for (int i = 0; i < flushOutDatas.length; i++) {
-            assertEquals(flushOutDatas[i], expectedResult[i]);
+            assertEquals(expectedResult[i], flushOutDatas[i]);
+        }
+    }
+
+    @Test
+    public void testPrintKeyValueMapperWithName() {
```

Comment:
Hmm, i'm ok with using test, but we already know it is a test case. Also using `should` helps to describe what the class should do.